v22.1/row-level-ttl.md

```diff
@@ -0,0 +1,323 @@
+
+| Description                                              | Option                                                                                                                                                                       |
+|----------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `ttl_expire_after` <a name="param-ttl-expire-after"></a> | The [interval](interval.html) when a TTL will expire. Default: '30 days'. Minimum value: '5 minutes'.                                                                        |
+| `ttl` <a name="param-ttl"></a>                           | Signifies if a TTL is active. Automatically set.                                                                                                                             |
+| `ttl_select_batch_size`                                  | How many rows to [select](select-clause.html) at one time from the set of expired rows. Default: 500. Minimum: 1.                                                            |
+| `ttl_delete_batch_size`                                  | How many rows to [delete](delete.html) at a time. Default: 100. Minimum: 1.                                                                                                  |
+| `ttl_delete_rate_limit`                                  | Maximum number of rows to be deleted per second (rate limit). Default: 0 (no limit).                                                                                         |
+| `ttl_range_concurrency`                                  | How many concurrent batches of expired rows are being worked on at a time. Default: `cpu_core_count`. Minimum: 1.                                                            |
+| `ttl_row_stats_poll_interval`                            | If set, counts rows and expired rows on the table to report as Prometheus metrics while the TTL job is running. Unset by default, meaning no stats are fetched and reported. |
+| `ttl_pause` <a name="param-ttl-pause"></a>               | If set, stops the TTL job from executing.                                                                                                                                    |
+| `ttl_job_cron` <a name="param-ttl-job-cron"></a>         | Frequency at which the TTL job runs, specified using [CRON syntax](https://cron.help).                                                                                       |
+| `ttl_automatic_column`                                   | If set, use the value of the `crdb_internal_expiration` hidden column. This is unset when `ttl_expiration_expression` is used.                                               |
```

Comment:
Unfortunately `ttl_expiration_expression` is not available yet, so it just appears and is mostly unconfigurable.

Reply:
Just deleted that second sentence for now.  No sense getting into it too much until later release, I think.  let me know if you disagree